README.md

```diff
@@ -42,3 +42,15 @@ in this repository's root directory.
 
 After running DocC, open the link that `docc` outputs
 to display a local preview in your browser.
+
+> Note:
+>
```

Comment:
If the explicit "Note" feels too heavy, maybe we don't need this in an aside/box at all.  I think it would also work to just include these two sentences at the end of the section.

Reply:
For me, this is okay, because I think it is some kind of supplementary information. And it is not particularly important for developers who are familiar with the context. (How to find `docc` executable)

Reply:
I think we should keep it in a note. You mentioned supplementary information, which reminds me that [Apple Style Guide](https://support.apple.com/guide/applestyleguide/n-apsgb2f4a0d6/1.0/web/1.0) calls for Note boxes for information that’s relevant to a topic but may not apply to all readers, or for a bit of information that’s tangential to a topic.